Userland/Libraries/LibCore/LocalSocket.cpp

```diff
@@ -14,6 +14,8 @@
 
 #if defined(__FreeBSD__)
 #    include <sys/ucred.h>
+#elif defined(__OpenBSD__)
+#    include <sys/socket.h>
```

Comment:
We already `#include <sys/socket.h>` above, line 13, why is this duplicate include also needed?

Reply:
It isn't. I missed removing that when I was mucking about with the creds/finding the pid